### ISSUE TITLE
Included Files dropdown doesn't show all included files

### ISSUE THREAD

**Issue by ERSUCC** [NetLogo/NetLogo#2319]
The Included Files dropdown only shows the files included by the current file, but I think it would be better to show the included files for the entire model.

**Comment by brandesNW** [comment]
I agree that this makes sense, but don’t think it is critical to implement.

--
Aaron Brandes, Software Developer
Center for Connected Learning and Computer-Based Modeling


From: Isaac Brown ***@***.***>
Date: Wednesday, September 4, 2024 at 6:44 PM
To: NetLogo/NetLogo ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [NetLogo/NetLogo] Included Files dropdown doesn't show all included files (Issue #2319)

The Included Files dropdown only shows the files included by the current file, but I think it would be better to show the included files for the entire model.

**Comment by ERSUCC** [comment]
Fixed in commit [08a5f37](https://github.com/NetLogo/NetLogo/commit/08a5f37fd6e33bd00696d765c4bd0585366285c9).
